grpc-client-runtime/build.gradle

```diff
@@ -15,6 +17,8 @@ dependencies {
     implementation libs.micronaut.buffer.netty
```

Comment:
Change all these `micronaut` dependencies to use the `mn` catalog, `mn.micronaut.runtime`, etc.